docs/advanced_configuration.md

```diff
@@ -90,4 +90,20 @@ Rack::Attack.blocklist('basic auth crackers') do |req|
     auth.credentials != [my_username, my_password]
   end
 end
-```
\ No newline at end of file
+```
+
+### Match Actions in Rails
+
+Instead of matching the URL with complex regex, it can be much easier to mach specific controller actions:
+
+```ruby
+Rack::Attack.safelist('unlimited requests') do |request|
+  safelist = [
+    'controller#action',
+    'another_controller#another_action'
+  ]
+  route = (Rails.application.routes.recognize_path request.url rescue {}) || {}
```

Comment:
Which is the potential exception being rescued here?

Reply:
It can raise `ActionController::RoutingError` and maybe other exceptions. And yes this code is working fine in production.